content/influxdb/v2.2/api-guide/iot-starter/python.md

```diff
@@ -0,0 +1,599 @@
+   3. [Create a configuration file for authentication with InfluxDB](#create-config.ini)
+3. [Create a virtual IoT device](#create-iot-virtual-device)
+4. [Develop with the API](#develop-with-the-api)
+   1. [Store virtual IoT device to InfluxDB using the API](#store-virtual-iot-device-to-influxdb)
+   2. [Query bucket for IoT device using the API](#query-bucket-for-device)
+   3. [Write telemetry data to InfluxDB using the API](#write-telemetry-data)
+   4. [Query telemetry data in InfluxDB using the API](#query-telemetry-data)
+5. [Connect the UI to the API](#connect-the-ui-to-the-api) 
+
+
+## Set up InfluxDB
+
+If you haven't already, 
+{{% cloud-only %}}[create an InfluxDB Cloud account](https://www.influxdata.com/products/influxdb-cloud/){{% /cloud-only %}}
+{{% oss-only %}}[install and set up InfluxDB OSS](https://www.influxdata.com/products/influxdb/){{% /oss-only %}}.
+
+### Authenticate with an InfluxDB API token
+
+For convenience in development, 
+[create an _All-Access_ token](/influxdb/v2.2/security/tokens/create-token/) 
+for your application. This grants your application full read and write 
+permissions on all resources within your InfluxDB organization.
+
+{{% note %}}
+
+For a production application, create and use a 
+{{% cloud-only %}}custom{{% /cloud-only %}}{{% oss-only %}}read-write{{% /oss-only %}} 
+token with minimal permissions and only use it with your application.
+
+{{% /note %}}
+
+## Introducing IoT Center
+
+The IoT Center architecture has four layers:
+
+- **InfluxDB API**: InfluxDB v2 API.
+- **IoT device**: Virtual or physical devices write IoT data to the InfluxDB API.
+- **IoT Center UI**: User interface sends requests to IoT Center server and renders views for the browser.
+- **IoT Center server**: Server and API receives requests from the UI, sends requests to InfluxDB,
+  and processes responses from InfluxDB.
+
+### Create IoT Center 
+
+Use [Flask](https://flask.palletsprojects.com/), a light-weight Python web 
+framework, and [Jinja](https://jinja.palletsprojects.com/), a templating engine,
+to create your sample IoT Center application. 
+
+1. Create and navigate into a new project directory.
+    ```bash
+    $ mkdir iotproject && cd $_
+    ```
+2. Create and activate a Python virtual environment for the new project.
+    ```bash
+    # Create a new virtual environment named "virtualenv"
+    # Python 3.8+
+    $ python -m venv virtualenv
+    # Activate the virtualenv (OS X & Linux)
+    $ source virtualenv/bin/activate
+    ```
+3. After activation completes, use the `pip` package installer (included with Python) to
+install Flask and Jinja. 
+    ```bash
+    $ pip install Flask
+    $ pip install Jinja
+    ```
+   
+### Install influxdb-client Package
+Use pip to install the influxdb-client package in your virtual environment.
+
+For more information about the package, see [the Python InfluxDB client package](https://pypi.org/project/influxdb-client/).
+```bash
```

Comment:
~~~suggestion
For more information about the package, see [the Python InfluxDB client package](https://pypi.org/project/influxdb-client/).

```bash
~~~